Neos.Neos/Classes/FrontendRouting/Projection/DocumentTypeClassification.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Neos.Neos package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\Neos\FrontendRouting\Projection;
+
+use Neos\ContentRepository\Core\NodeType\NodeTypeManager;
+use Neos\ContentRepository\Core\NodeType\NodeTypeName;
+use Neos\Neos\Domain\Service\NodeTypeNameFactory;
+
+enum DocumentTypeClassification: string
```

Comment:
btw: this does not have to be a backed enum IMO

Reply:
adjusted